### ISSUE TITLE
About the BPCC dataset

### ISSUE THREAD

**Issue by rose-jinyang** [AI4Bharat/IndicTrans2#125]
Hello
How are you?
I am going to use the BPCC dataset for fine-tuning a model.
But there are several sub-datasets in the BPCC dataset (https://huggingface.co/datasets/ai4bharat/BPCC).
Should I merge all the sub-sets?
Or, May I use ONLY the largest sub-set (nllb-filtered)?

![Image](https://github.com/user-attachments/assets/3868960b-749d-4657-a990-52db32101676)

**Comment by PranjalChitale** [comment]
For fine-tuning pre-trained translation models, the high-quality human-annotated Bpcc-Seed-Latest subset would offer greater data efficiency alongside strong performance gains.

However, in case you want to train a model from scratch, you can merge all data subsets and train jointly to replicate the setup used for training IndicTrans2.
